template/script.lisp

```diff
@@ -0,0 +1,9 @@
+(require '#:asdf)
+
+(asdf:load-system '#:{{LIBRARY}})
```

Comment:
it's now preferred to use lowercase string names for ASDF system designators I believe